h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -999,4 +1000,32 @@ public void printOutColumnsMetadata(Frame fr) {
     }
   }
 
+  protected static RowData toRowData(Frame fr, String[] columns, long row) {
```

Comment:
I am surprised why it is here right now. I have used this for loop in test many times already. :+1: ;-)